typescript/src/electrum.ts

```diff
@@ -35,7 +35,7 @@ export interface Credentials {
 /**
  * Additional options used by the Electrum server.
  */
-type Options = object
+export type ClientOptions = object
```

Comment:
We need to update the usage of this type in the `Client` class as well.

Reply:
Sorry, wanted to do it too fast 🤦‍♂️ Fixed in e7b1090c82638ab4009e27cbaeff75e11e76f4d4